openfisca_uk/variables/hmrc/tax_credits/working_tax_credit/elements/basic.py

```diff
@@ -0,0 +1,14 @@
+from openfisca_uk.model_api import *
+
+
+class wtc_basic_element(Variable):
```

Comment:
If this parameter-only variable exists to work with the list parameters in aggregation, could you make a comment somewhere about that?